Tests/Unit/Module/STIG.DomainName.tests.ps1

```diff
@@ -0,0 +1,92 @@
+#region Header
+. $PSScriptRoot\.tests.header.ps1
+#endregion
+
+$domainName = 'Domain.test'
+$forestName = 'Forest.test'
+$domainDistinguished = "DC={0},DC={1}" -f $domainName.Split(".")
+$forestDistinguished = "DC={0},DC={1}" -f $forestName.Split(".")
+$domainDistinguishedPart1 = "DC=Domain"
+$domainDistinguishedPart2 = "DC=Domain,DC=Test"
+$parts = @("Domain","Test")
+
+Describe 'Get-DomainName' {
+        # Test parameter validity -OutputPath
+        It 'Should return Domain FQDN' {
+            Get-DomainName -DomainName $DomainName -Format 'FQDN' | Should -Match $domainName
+        }
+
+        It 'Should return Forest FQDN' {
+            Get-DomainName -ForestName $ForestName  -Format 'FQDN' | Should -Match $forestName
+        }
+
+        It 'Should return Domain NetbiosName' {
+            Get-DomainName -DomainName $DomainName -Format 'NetbiosName' | Should -Match ($domainName.Split("."))[0]
+        }
+
+        It 'Should return Forest NetbiosName' {
+            Get-DomainName -ForestName $ForestName  -Format 'NetbiosName' | Should -Match ($forestName.Split("."))[0]
+        }
+
+        It 'Should return Domain DistinguishedName' {
+            Get-DomainName -DomainName $DomainName -Format 'DistinguishedName' | Should Match $domainDistinguished
```

Comment:
To be consistent with the statements above, the "Should Match..." should be "Should -Match" and line 36.

Reply:
Updated